src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -239,16 +259,16 @@ private void attachCaseByCcdId(
         String exceptionRecordJurisdiction,
         String targetCaseCcdRef,
         List<Map<String, Object>> exceptionRecordDocuments,
-        Long exceptionRecordId,
+        CaseDetails exceptionRecord,
```

Comment:
if entire exception record is passed then maybe`exceptionRecordDocuments` doesn't need to be passed separately?